src/debug/dict.py

```diff
@@ -46,7 +46,7 @@ def __delitem__(self, k: KT) -> None:
         self._history.add("delitem", k)
         del self._dict[k]
 
-    def __getitem__(self, k: KT) -> VT_co:
+    def __getitem__(self, k: KT) -> VT:
```

Comment:
Why this change? Do you know what covariance is? I'm nearly certain this was correct before, because we may be returning a type that is covariant in VT, not necessarily an instance of VT itself.